### ISSUE TITLE
Travis is dying

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/conda-forge.github.io#1521]
Aside from the previous instabilities and random timeouts, it now seems to fail even trivial (& fundamental) tasks like downloading binaries consistently (4x in a row over the course of 24h in https://github.com/conda-forge/openblas-feedstock/pull/122, each time across all 4 jobs in the build)
```
Downloading https://github.com/xianyi/OpenBLAS/archive/v0.3.18.tar.gz
Error: HTTP 000 CONNECTION FAILED for url <https://github.com/xianyi/OpenBLAS/archive/v0.3.18.tar.gz>
```

I don't know when this will resolve itself, but just thought I'd open an issue to discuss the situation (e.g. move all PPC builds to azure?).

**Comment by jakirkham** [comment]
There was at least one recent change for Travis ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/167 ). Maybe more are needed? Honestly would bring this up on [the gitter channel]( https://gitter.im/conda-forge-ppc64le/Lobby )

**Comment by h-vetinari** [comment]
To be honest, after a year of flakiness, I have very limited patience left for debugging Travis... For the feedstocks I maintain more or less by myself, I'll see if emulation through azure works, otherwise I'll start dropping PPC.

**Comment by h-vetinari** [comment]
I'm wondering if github is blocking travis from downloading sources. I can't believe that this is so consistently failing (all jobs I've seen recently, and every run therein) just due to flaky networks.

**Comment by jakirkham** [comment]
Yeah this is what I'm wondering as well

**Comment by nehaljwani** [comment]
Is this related? https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/168

**Comment by mbargull** [comment]
Due to the [cloud.drone.io breakage](https://discourse.drone.io/t/builds-marked-as-error-and-new-builds-no-longer-starting/10011/3) Travis is now also used for `linux-aarch64` if one uses `native` build platform in `conda-forge.yml`. (That is just until we have a working usable alternative service running for ARM builds again.)

Now, apparently for that platform there is some issue (misconfiguration?) with seccomp, which leads to non-working `sudo` in the containers (used for `sudo yum install` for `yum_requirements.txt`) :unamused:. We encountered this on https://github.com/conda-forge/python-feedstock/pull/523 and https://github.com/conda-forge/python-feedstock/pull/523/commits/0f13bea3b35e8587d2327ed44178d5937a189a9b shows that with `seccomp=unconfined` that issue is "resolved".
Meaning, one could dig into/fix the seccomp profile on Travis' ARM workers to add a proper fix.
But at least the `python-feedstock` build was either way killed after some minutes because `The job exceeded the maximum log length, and has been terminated.` (i.e., `conda-build`'s `$PREFIX`/`$BUILD_PREFIX`/`$SRC_DIR` replacement would need some fixes/be more aggressive -- running on `stderr` too and support `outputs/script` too). So we'd to turn multiple knobs for the (hopefully) temporary usage of Travis for ARM builds.
My recommendation for now is to use the emulated builds on Azure wherever possible.

**Comment by h-vetinari** [comment]
I've seen various feedstocks where jobs on travis silently aren't being started at all anymore, giving the impression the CI is green while a chunk of the matrix was never built. This is arguably even worse than failing jobs (https://github.com/conda-forge/status/issues/185 didn't get fixed for month either), c.f. also https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6595.

All in all:


![Image](https://github.com/user-attachments/assets/78f1fa8b-0b04-4220-bac8-cd4e424103d5)

**Comment by beckermr** [comment]
Yeah I'm in agreement here. Travis seems to have almost completely rotted away. I'm sick of looking at the status issue. We should remove it totally and push any jobs that cannot cross-compile into emulation, possibly on the long cpu queue on the gpu server if they need more than six hours.

**Comment by danielhollas** [comment]
> I've seen various feedstocks where jobs on travis silently aren't being started at all anymore, giving the impression the CI is green while a chunk of the matrix was never built. 

Can confirm, just noticed that we're missing a linux_aarch64 build on latest release of rabbitmq-server, so easy to miss when CI is green. 😦 

Since I haven't seen this mentioned, after logging in to Travis CI and trying to restart a job, I saw the following message:

> We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution.

![Image](https://github.com/user-attachments/assets/b5ed54ea-d12e-4eea-abda-a7579853ce2a)

**Comment by beckermr** [comment]
Thanks for this note! This seems like something we might be able to take action on with travis support. @conda-forge/core does anyone have any open or recent tickets with them, or should we open a new one?
